pgjdbc/src/main/java/org/postgresql/jdbc/TimestampUtils.java

```diff
@@ -948,16 +943,16 @@ public String toString(OffsetTime offsetTime) {
 
   public String toString(OffsetDateTime offsetDateTime) {
     try (ResourceLock ignore = lock.obtain()) {
-      if (offsetDateTime.isAfter(MAX_OFFSET_DATETIME)) {
+      if (offsetDateTime.equals(OffsetDateTime.MAX)) {
```

Comment:
This seems odd. What do we do with a date that is after `MAX_OFFSET_DATETIME` ?

Reply:
We pass the date to the DB, and it either accepts or rejects it just like all the other dates.

Reply:
finding out what postgres' definition of infinity is is difficult. Makes sense